### ISSUE TITLE
Twitter followers Badge seems to be broken

### ISSUE THREAD

**Issue by joergrech** [badges/shields#8969]
### Are you experiencing an issue with...

shields.io

### 🐞 Description

Badges with Twitter followers are not working anymore. If you try the following link you see that the number of followers is not shown anymore:

`https://img.shields.io/twitter/follow/Shields_io?logo=Twitter&label=Followers&style=for-the-badge`

Other social networks like Reddit Karma, Github Followers, etc. seems to work.

### 🔗 Link to the badge

https://img.shields.io/twitter/follow/Shields_io?logo=Twitter&label=Followers&style=for-the-badge

### 💡 Possible Solution

I've heard that Twitter changed their API - at least IFTTT send the following in an email in Febrauary:

_Starting Monday, February 13th, 2023, Twitter will no longer support free access to their API. As a result, we expect that any Applet that connects with Twitter will stop working. You can learn more about Twitter's upcoming API change [on Twitter](https://twitter.com/TwitterDev/status/1621026986784337922)._

**Comment by chris48s** [comment]
See
- https://github.com/badges/shields/issues/8837
- https://github.com/badges/shields/pull/8842
For more details of why this has been removed.

You are right that the way we've done this does make the non-social forms of the fallback look  a bit strange.

**Comment by calebcartwright** [comment]
Is there any action to be taken here? I agree these look odd, but given how big of a special case we've employed for the badge (with our hand forced by the upstream Twitter changes) this has essentially been reduced a static badge, and anyone that really wants the "equivalent" badge without the funky message portion could, and I'd suggest should, do so directly with [static badge builder](https://shields.io/#your-badge)

**Comment by PyvesB** [comment]
Agreed with Caleb, I don't think there's anything actionnable here. Will go ahead and close the issue.
